src/Compilers/VisualBasic/Test/Syntax/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.CodeAnalysis.Test.Resources.Proprietary" version="1.0.0-rc3-20150528-03" targetFramework="net45" />
-  <package id="xunit" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.abstractions" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.assert" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.core" version="2.0.0-alpha-build2576" targetFramework="net45" />
+  <package id="xunit" version="2.1.0-beta2-build2981" targetFramework="net45" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
```

Comment:
Possibly, but as per my comment above, it should really be possible to remove the xUnit references from all of these packages.config files and move them to the LCD assembly (I think this is TestUtilities.csproj). The packages would then be downloaded/synced as a dependency of each of the other projects.